tests/new/test.scala

```diff
@@ -1,3 +1 @@
-trait T {
-  object O
-}
+inline def foo = 1
```

Comment:
This seems to have been committed by mistake?

Reply:
yes indeed